EdFi.Ods.Utilities.Migration/Scripts/02 Upgrade/v34_to_v50/01 Structure/1010-AddNewDescriptorValuesForV50.sql

```diff
@@ -0,0 +1,44 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/QuestionFormDescriptor', 'Checkbox', 'Checkbox', 'Checkbox');
```

Comment:
There's still quite a few missing descriptor values here. There should be values for all the tables you've added in your 1000-AddNewDescriptorTablesForV50.sql script. I think there was 50+. You can see some of them here for example: https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Standard/blob/v3.2.0-c/Descriptors/QuestionFormDescriptor.xml. Pulling them from a 5.0.0 database is probably easier than pulling them from the xml though.

Reply:
Yes I missed it and now added missed one  .  But in total 7 tables in 1000-AddNewDescriptorTablesForV50.sql script.